def main():
    try:
        num = 11/0
    except Exception as err:
        print("fuck u", err)

main()
